zerver/tests/test_invite.py

```diff
@@ -1627,15 +1627,18 @@ def test_successful_get_open_invitations(self) -> None:
         hamlet = self.example_user("hamlet")
         othello = self.example_user("othello")
 
-        streams = []
+        stream_objects = []
```

Comment:
I don't see any reason to rename `streams` to `stream_objects` here in this commit. We anyways have `stream_ids` as the second list which is enough to differentiate between the two. We can avoid such changes and can do so in a separte rename commit if required.